GGProject/scripts/adminOptions.py

```diff
@@ -1,11 +1,12 @@
 import requests as req
 import ldap
 from django_auth_ldap.backend import LDAPBackend
+from GreeterGuru import settings
 
 
 def populate():
 
-    host = "192.168.200.128"		   # Server’s name/IP address
+    host = settings.SERVER_NAME		   # Server’s name/IP address
     username = "internship\\Administrator" # Any user’s login name on the server
```

Comment:
Does it make sense to have the username/ password be accessible from the "settings.py" similar to the above line?